Public/Src/Cache/ContentStore/Hashing/ContentHashWithLastAccessTime.cs

```diff
@@ -13,14 +13,14 @@ namespace BuildXL.Cache.ContentStore.Hashing
         /// <nodoc />
         public ContentHashWithLastAccessTime(ContentHash contentHash, DateTime lastAccessTime)
         {
-            Hash = contentHash;
+            ContentHash = contentHash;
             LastAccessTime = lastAccessTime;
         }
 
         /// <summary>
         ///     Gets the content hash member.
         /// </summary>
-        public ContentHash Hash { get; }
+        public ContentHash ContentHash { get; }

```

Comment:
This may be a breaking change.

Is there any specific reason for this rename?

Reply:
Seemed like an unnecessary inconsistency wrt to the other ContentHashWith*. Didn't know it could be breaking. Just rolled it back